README.md

```diff
@@ -1,24 +1,80 @@
-**STILL IN DEVELOPMENT, NOT FIT FOR GENERAL CONSUMPTION, DO NOT USE IF NOT A DEVELOPER!**
-
 # Slic3r plugin for OctoPrint
 
-**WARNING** Not functional yet since slic3r's return codes are behaving unexpectedly. WIP for others to take a look at
```

Comment:
Then we should leave this warning in for now. If the binary still behaves as erratically, that's a problem. I can try to take a look at that, but I'm still having my hands full with all that self employment and crowd funding stuff so that this might take a while.